libhb/common.c

```diff
@@ -314,6 +318,15 @@ static int hb_video_encoder_is_enabled(int encoder, int disable_hardware)
             case HB_VCODEC_FFMPEG_VT_H265:
                 return hb_vt_h265_is_available();
 #endif
+
+#if HB_PROJECT_FEATURE_MF
+            // TODO: Try to instantiate a throwaway encoder to see if a suitable MediaFoundation encoder can be found?
```

Comment:
@mstorsjo Is this something you are or will be working on?

Reply:
Not directly planning on it, no, it wasn't really in scope for what I did when working on this branch originaly.

Reply:
Okay, thanks for clarifying.